rfcs/detecting_sensitive_textual_content/20230309-project_proposal_detecting_sensitive_textual_content.md

```diff
@@ -0,0 +1,379 @@
+# Project Proposal: Detecting sensitive textual content
+
+## Reviewers
+
+- [ ] @stacimc or @aetherunbound
+- [ ] @krysal
+
+## Project summary
+
+Detect results with sensitive textual content by checking their textual content
```

Comment:
Would adding the level where detection takes place be useful here? "Detect in the API"?

Reply:
I tried to avoid implying any specific implementation details in this summary and to just cover the broad strokes of what the feature should entail that would otherwise be duplicated in the Requirements section below.